src/index.css

```
@@ -10,16 +10,75 @@ body {
   margin: 0;
 }
 
-.esri-widget,
+@layer base {
+  body {
+    font-family:
+      -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+      'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+      sans-serif;
+  }
+
+  code {
+    font-family:
+      source-code-pro, Menlo, Monaco, Consolas, 'Courier New', monospace;
+  }
+
+  h1 {
+    @apply text-3xl font-bold md:text-4xl;
+  }
+  h2 {
+    @apply text-2xl font-bold md:text-3xl;
+  }
+  h3 {
+    @apply text-xl font-bold md:text-2xl;
+  }
+  h4 {
+    @apply text-lg font-bold md:text-xl;
+  }
+  h5 {
+    @apply text-base font-bold md:text-lg;
+  }
+  h6 {
+    @apply text-sm font-bold md:text-base;
+  }
+
+  @font-face {
+    font-family: 'SourceSansPro-Regular';
+    src: url('/fonts/SourceSans3-Regular.otf.woff2') format('woff2');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+  }
+
+  @font-face {
+    font-family: 'SourceSansPro-Black';
+    src: url('/fonts/SourceSans3-Black.otf.woff2') format('woff2');
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
+  }
+}
+
+/* this makes the focus border match the rounded size of the container button */
 calcite-button {
+  --calcite-button-corner-radius: 9999px;
+}
+
+.esri-component,
+.esri-widget calcite-button {
   @apply rounded-full;
 }
 
 .esri-ui-top-right {
   @apply flex-row space-x-3;
 }
 
-#filter-switch {
-  --calcite-color-brand: var(--calcite-color-border-2);
-  --calcite-color-brand-hover: var(--calcite-color-border-2);
+.esri-view-surface::after {
+  outline-color: transparent !important;
+}
+
+.esri-basemap-toggle calcite-button,
+.esri-basemap-toggle__container,
+.esri-basemap-toggle__image {
+  @apply !size-[32px] !rounded-full;
```

you can divide everything by 4 in tw 3 to use the standard scale.

```suggestion
  @apply !size-8 !rounded-full;
```